### PR TITLE
Use outer type if present then inner message level.

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

File: src/test/kotlin/app/batch/HBaseReaderTest.kt [hunk 1]
```diff
@@ -111,6 +111,67 @@ class HBaseReaderTest {
             expected.toString(), actual.toString())
     }
 
+    @Test
+    fun testInnerType() {
```
Comment (steveburton4) [inline src/test/kotlin/app/batch/HBaseReaderTest.kt]: We need a test for when the @type field is empty and one for when the @type field doesn't exist.

Comment (DanielChicot) [reply src/test/kotlin/app/batch/HBaseReaderTest.kt]: Done.
